src/__test__/Details.test.js

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom';
+import { Provider } from 'react-redux';
+import Details from '../components/Details';
+import store from '../redux/configStore';
```

Comment:
- [ ] I kindly suggest that you correct the letter of your r in redux to Capital and the letter c in components to also capital because your folders names are in Capital letter and this will cause your tests to fail if ran from the root folder. Please make changes to all tests files affected.
![image](https://user-images.githubusercontent.com/52934128/175771908-74cf0b6f-b44a-4167-a759-9a6c2a3a60dc.png)

Reply:
@topeogunleye Thanks for pointing out the issues. I will fix these right away 🙏🏾